### ISSUE TITLE
Unknown URL hash breaks the javascript execution

### ISSUE THREAD

**Issue by jonatasoliveira** [alvarotrigo/fullPage.js#2284]
### Description

When using a hash in the URL that is not defined in the HTML as a section ID this exception is raised:

```
jquery.fullPage.js:1392 Uncaught TypeError: Cannot read property 'top' of undefined
```

### Link to isolated reproduction with no external CSS / JS

http://alvarotrigo.com/fullPage/examples/responsive.html#token=1234567890

### Versions 
Chrome for Mac Version 53.0.2785.116 (64-bit)

**Comment by alvarotrigo** [comment]
Thanks for reporting it!!

Does it means any Javascript underneath won't be interpreted ?

**Comment by jonatasoliveira** [comment]
@alvarotrigo, yes.

I've done a fix in the code, do you think it's appropriated to create a Pull Request?

**Comment by alvarotrigo** [comment]
> I've done a fix in the code, do you think it's appropriated to create a Pull Request?

Sure! Make sure to do it in the `dev` branch as stated in the pull request skeleton. Then I'll review it.

**Comment by CorpusCallosum** [comment]
Hi! I am having this same issue. Is there a fix available? Thanks!

**Comment by alvarotrigo** [comment]
@jonatasoliveira planning to commit the pull request?

**Comment by alvarotrigo** [comment]
Fixed on dev.

**Comment by jonatasoliveira** [comment]
Sorry for the delay. @alvarotrigo I saw that you fixed with a different approach, thank for that!

**Comment by alvarotrigo** [comment]
Solved in 2.8.7
